amespahdbpythonsuite/fitted.py

```diff
@@ -204,12 +210,20 @@ def plot(self, **keywords) -> None:
             else:
```

Comment:
Units should already be in self.observation.unit / self.observation.unit; self.observation is a Spectrum1D object.